frontends/api/src/hooks/learningPaths/queries.ts

```diff
@@ -24,21 +24,21 @@ const learningPathKeys = {
 
 const learningPathQueries = {
   list: (params: ListRequest) =>
-    ({
+    queryOptions({
```

Comment:
See https://tkdodo.eu/blog/the-query-options-api#queryoptions

This is just a TS helper.